### PR TITLE
EVENT-425

### PR DESCRIPTION
**Pull request by LaelWatkins**
This change will rename/relabel Registration Types to Registrant Types throughout the tool.  This is just text changes.  The user has reviewed and approved.

https://jira.cru.org/browse/EVENT-425

### CODE REVIEW

File: app/scripts/controllers/eventDetails.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ angular.module('confRegistrationWebApp')
       } else {
         $scope.notify = {
           class: 'alert-danger',
-          message: $sce.trustAsHtml('You must have at least one registration type per event.')
+          message: $sce.trustAsHtml('You must have at least one Registrant type per event.')
```
Comment (adammeyer) [inline app/scripts/controllers/eventDetails.js]: This supposed to be capitol here?  I'm not sure.

**Comment by adammeyer** [comment]
👍

File: app/scripts/controllers/eventDetails.js (hunk 1 above)
Comment (LaelWatkins) [inline app/scripts/controllers/eventDetails.js]: it shouldn't have been.  Its fixed.

**Comment by LaelWatkins** [comment]
Thanks I'll merge later this morning.

**Pull request merged**